### PR TITLE
Chevron should be updated when using RTL

### PR DESCRIPTION
**Pull request by ehab-al-cko**
[Jira ticket](https://checkout.atlassian.net/browse/PIMOB-1281?atlOrigin=eyJpIjoiOWZmM2IwNDMxNjgyNDAxZmExYmZjMzRhMzQ1MDE1OTUiLCJwIjoiaiJ9)

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: Tests/UI/New UI/BillingForm/View/BillingFormSummaryViewTests.swift [hunk 1]
```diff
@@ -46,7 +46,7 @@ class BillingFormSummaryViewTests: XCTestCase {
     }
 
     func testStyleIconImageView() {
-        XCTAssertEqual(view.imageContainerView?.imageView?.image, style.button.image)
+        XCTAssertEqual(view.imageContainerView?.imageView?.image, style.button.image?.imageFlippedForRightToLeftLayoutDirection())
```
Comment (deepesh-vasthimal-cko) [inline Tests/UI/New UI/BillingForm/View/BillingFormSummaryViewTests.swift:49]: Question : Can you compare 2 images ? this way. Will this even be a valid test.

Comment (ehab-al-cko) [reply Tests/UI/New UI/BillingForm/View/BillingFormSummaryViewTests.swift:49]: I think, It's valid test. this test is against the the provided image style.

File: Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift [hunk 2]
```diff
@@ -20,7 +20,11 @@ class ImageContainerView: UIView {
     }
 
     func update(with image: UIImage?, tintColor: UIColor? = nil) {
-      imageView?.image = image?.withRenderingMode(tintColor == nil ? .alwaysOriginal : .alwaysTemplate)
+      imageView?.image = image?
+                        .withRenderingMode(tintColor == nil ?
+                                            .alwaysOriginal :
+                                            .alwaysTemplate)
+                        .imageFlippedForRightToLeftLayoutDirection()
```
[the remark this answers is not in the document]
Comment (ehab-al-cko) [reply Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift]: Fixed 
Simulator Screen Shot - iPhone 13 - 2022-07-07 at 14 48 35

File: Source/UI/NewUI/CommonUI/View/Component/SelectionButtonView.swift [hunk 3]
```diff
@@ -6,7 +6,7 @@ protocol SelectionButtonViewDelegate: AnyObject {
 
 class SelectionButtonView: UIView {
     weak var delegate: SelectionButtonViewDelegate?
-    private var style: CellButtonStyle?
+    private(set) var style: CellButtonStyle?
```
Comment (alex-ioja-yang-cko) [inline Source/UI/NewUI/CommonUI/View/Component/SelectionButtonView.swift:9]: This was already being tested successfully previously without while being private, or was something being missed?

Comment (ehab-al-cko) [reply Source/UI/NewUI/CommonUI/View/Component/SelectionButtonView.swift:9]: it's needed for test case

File: Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift [hunk 4]
```diff
@@ -14,25 +14,25 @@ class BillingFormButtonViewTests: XCTestCase {
...
-        XCTAssertEqual(view.image?.imageView?.image, style.button.image)
-        XCTAssertEqual(view.image?.imageView?.tintColor, style.button.disabledTintColor)
+        XCTAssertEqual(view.imageContainerView.imageView?.image, view.style?.button.image)
+        XCTAssertEqual(view.imageContainerView.imageView?.tintColor, style.button.disabledTintColor)
     }
 
     func testErrorStyle() throws {
-        let isHidden = try XCTUnwrap(view.errorView?.isHidden)
+        let isHidden = try XCTUnwrap(view.errorView.isHidden)
         XCTAssertTrue(isHidden)
```
Comment (harry-brown-cko) [inline Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift:36]:
```suggestion
        XCTAssertTrue(view.errorView.isHidden)
```

method should also no longer throw

File: Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift [hunk 5]
```diff
@@ -14,25 +14,25 @@ class BillingFormButtonViewTests: XCTestCase {
...
     }
 
     func testImageStyle() {
-        XCTAssertEqual(view.image?.imageView?.image, style.button.image)
-        XCTAssertEqual(view.image?.imageView?.tintColor, style.button.disabledTintColor)
+        XCTAssertEqual(view.imageContainerView.imageView?.image, view.style?.button.image)
+        XCTAssertEqual(view.imageContainerView.imageView?.tintColor, style.button.disabledTintColor)
     }
 
     func testErrorStyle() throws {
```
Comment (harry-brown-cko) [inline Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift:34]:
```suggestion
    func testErrorStyle() {
```

**Review by alex-ioja-yang-cko — APPROVED** [review]

**Pull request merged**